### DISCUSSION TITLE
Used AssetLib to upgrade from v1.3 to v1.4.2 - but don't see new functionality

### DISCUSSION THREAD

**Discussion by sshearer** [dialogic-godot/dialogic#923] [Q&A] [answered]
Hi, I've seen the v1.4 video and am excited to (finally) explore the new UI & features :)  

After doing the upgrade I see a number of folders & files on my drive have been updated in my project, but I don't see the new UI and in my timeline it still shows the previous version number.  I have restarted godot a couple times and tried saving a minor change to the timeline, but still don't see the new UI.  I've searched the Discussions, but didn't see anything helpful.

Can you please tell me what else I need to do, or tell me where to find what instructions ?

Thank you,
Steven.

**Comment by coppolaemilio** [comment, marked as answer]
Hey, unfortunately the update is a bit tricky to execute. 
First of all do a backup of everything so you don't accidentally delete anything from your project.
Disable Dialogic from the project settings, then remove the old dialogic version by removing the `/addons/dialogic`directory.
Now that everything is gone you can extract the new version on `/addons/dialogic` and enable it again from the project settings.
If everything goes right and nothing exploded while doing that you should be able to continue working on your project using the new version.

Hope that helps!

**Reply by sshearer** [reply]
That worked :)  Thanks very much.  I recommend the installation notes be updated with the above :)
